recipes/autoconf/all/conanfile.py

```diff
@@ -21,6 +21,11 @@ def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
 
+    def requirements(self):
+        if self.settings.os_build != "Windows":
+            #the Conan m4 package currently doesn't work correctly with autoconf
```

Comment:
Why the comment? What is the issue?

Reply:
This problem:
https://github.com/conan-io/conan-center-index/pull/2103#issuecomment-653470881

We don't know yet whether the problem lies with autoconf, m4, windows bash paths, ...